apps/oc_chef_wm/src/oc_chef_wm_base.erl

```diff
@@ -1123,6 +1161,7 @@ maybe_authz_id(B) ->
     B.
 
 
+
```

Comment:
lol is this like that scene in goonies where I find chester copperpot's wallet?

![](https://lh3.googleusercontent.com/-iSLYeiTsdLs/UK0R1gZLKzI/AAAAAAAAACo/mqnw5qF2_M4/s630/CopperPot.jpg)